R/run_stress_test.R

```diff
@@ -4,7 +4,10 @@
 #' understand sensitivities of the scenarios, in which case the user may pass a
 #' vector of values to one (and only one) of the detail arguments. This will
 #' result in running the analysis multiple times in a row with the argument
-#' varied. NOTE: argument `asset_type` is not iterateable.
+#' varied.
+#' NOTE: argument `asset_type` is not iterateable.
+#' NOTE: if `return_results` is TRUE results will not be written to `output
```

Comment:
Does this mean that output path becomes optional? Do we check if at least one is given? Or do we need both arguments regardless? Maybe something we need to be extra clear about  with the users

Reply:
if return_results is TRUE output path does not need to be provided. Adjsuting.

Reply:
actually,  I have to correct myself. even if result are returned we write log files to output paths, so path needs to be provided. Guess that couls be emphasised in vignette.

Reply:
okay